models/upload.go

```diff
@@ -36,7 +37,7 @@ type Upload struct {
 	FileNameLowerCase  string `gorm:"type:varchar(255)"`
 	FileNameUpperCase  string `gorm:"type:varchar(255)"`
 	Extension          string `gorm:"type:varchar(255)"`
-	Size               int64  `gorm:"type:integer"` // upload size in bytes
+	Size               int64  `gorm:"type:bigint"` // upload size in bytes
```

Comment:
Why make this change?

Reply:
not sure, but CI builds started failing without this. Even though the `int64` golang type is enough to hold the data we need, postgresql logs were showing errors and overflow problems until i made this change.

Reply:
I see, go to know it doesn't cause problems with existing data.

Reply:
yea, i've been trying it out in dev and i dont think there's any problems as long as you do a migration